openml/datasets/functions.py

```diff
@@ -333,27 +333,40 @@ def _load_features_from_file(features_file: str) -> Dict:
         return xml_dict["oml:data_features"]
 
 
-def check_datasets_active(dataset_ids: List[int]) -> Dict[int, bool]:
+def check_datasets_active(
+    dataset_ids: List[int],
+    raise_error_if_not_exist: bool = True,
+) -> Dict[int, bool]:
     """
     Check if the dataset ids provided are active.
 
+    Raises an error if a dataset_id in the given list
+    of dataset_ids does not exist on the server.
+
     Parameters
     ----------
     dataset_ids : List[int]
         A list of integers representing dataset ids.
+    raise_error_if_not_exist : bool, optional (default=True)
```

Comment:
```suggestion
    raise_error_if_not_exist : bool (default=True)
```

It's not `optional`, `optional` means it can take a `None` value (as opposed to having a default value).

Reply:
@PGijsbers I also thought the same. However, going up I noticed this:
https://github.com/openml/openml-python/blob/e84cdf928b8de713ce0cc2a528f9f675025b783e/openml/datasets/functions.py#L207
So I tried to be consistent, I will change it.

Reply:
You'd be correct to point out that annotation is wrong too :)

Reply:
I will post an issue regarding that, since I also observed a function that had no documentation at the module.